ci/long_running_tests/config.yaml

```diff
@@ -14,48 +14,35 @@ auth:
 
 head_node:
     InstanceType: m5.xlarge
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI 7/1/19
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
 worker_nodes:
     InstanceType: m5.large
-    ImageId: ami-0def3275  # Default Ubuntu 16.04 AMI.
-
-    # Set primary volume to 25 GiB
+    ImageId: ami-0888a3b5189309429  # DLAMI 7/1/19
     BlockDeviceMappings:
         - DeviceName: /dev/sda1
           Ebs:
-              VolumeSize: 50
+              VolumeSize: 150
 
     # Run workers on spot by default. Comment this out to use on-demand.
     InstanceMarketOptions:
         MarketType: spot
 
 # List of shell commands to run to set up nodes.
 setup_commands:
-    - sudo apt-get update
-    - sudo apt-get install -y build-essential curl unzip
-    # Install Anaconda.
-    - wget https://repo.continuum.io/archive/Anaconda3-5.0.1-Linux-x86_64.sh || true
-    - bash Anaconda3-5.0.1-Linux-x86_64.sh -b -p $HOME/anaconda3 || true
-    - echo 'export PATH="$HOME/anaconda3/bin:$PATH"' >> ~/.bashrc
-    - echo 'termcapinfo xterm* ti@:te@' >> ~/.screenrc
-    # Some Python dependencies.
-    - pip install boto3==1.4.8 cython==0.29.0
-    # Uncomment the following if you wish to install Ray instead.
+    # Install nightly Ray wheels.
+    - source activate tensorflow_p36 && pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/<<<RAY_BRANCH>>>/<<<RAY_COMMIT>>>/ray-<<<RAY_VERSION>>>-cp36-cp36m-manylinux1_x86_64.whl
+    - source activate tensorflow_p36 && pip install ray[rllib] ray[debug] gym[atari]
```

Comment:
Do you know if `pip install ray[rllib]` will do the right thing? E.g., does it get the info from the locally installed Ray or from PyPI?

Reply:
It will get the right dependencies and ignore the local ray.